### ISSUE TITLE
issue with package "carbon" in eclipse

### ISSUE THREAD

**Issue by NeverGivinUp** [ZieIony/Carbon#79]
I'm using Eclipse. It does not accept the package name "carbon", as defined in the android manifest xml file. When I changed the package structure to "com.carbon" Eclipse was happy. It's a pain to do every time I update, though. And right now I can't as I can't get the full 0.7.4 source with fitting resources from GitHub. 
Please consider renaming the base package to "com.carbon" or "org.carbon" or "org.zielony.carbon" or anything with a dot.

**Comment by ZieIony** [comment]
Changing the package name is not possible right now, because it would break builds of the people using Carbon with the current package name. Maybe you could try using a Maven/Gradle plugin for Eclipse and use AAR packages.

**Comment by NeverGivinUp** [comment]
Hi Zielony,

I tried using Gradle for Eclipse and failed miserably. Now I'm trying to use Android Studio. I've imported my project and replaced dependencies with a link to Maven Central. I do get an error, that I cannot explain: 

```
F:\Softwareentwicklung\StudioProjects\TestApplication\app\build\intermediates\res\debug\drawable-hdpi-v4\ic_launcher.png: error: Duplicate file.
F:\Softwareentwicklung\StudioProjects\TestApplication\app\build\intermediates\res\debug\drawable-hdpi\ic_launcher.png: Original is here. The version qualifier may be implied.
```

I tested with a test project and the error appears only if I set a dependency to Carbon and have my own launcher drawables present. Any hints on how I can get around this?

Many thanks!
Navid

**Comment by ZieIony** [comment]
The reason is very simple - Carbon has these files as well, so there are duplicates. This is obviously wrong - libraries shouldn't use so simple file names and/or contain launcher icons. I will fix this today before 0.8.0 official release. Thanks for reporting this!

**Comment by ZieIony** [comment]
fixed in 0.8.0
